### ISSUE TITLE
UnpicklingError: invalid load key, '<'.

### ISSUE THREAD

**Issue by iuria21** [chsasank/ATIS.keras#4]
Hi, I'm having this error when loading the data, and I can't solve it:
```
UnpicklingError                           Traceback (most recent call last)
<ipython-input-13-d6cdf9258128> in <module>()
      2 import data.load
      3 
----> 4 train_set, valid_set, dicts = data.load.atisfull()
      5 w2idx, labels2idx = dicts['words2idx'], dicts['labels2idx']
      6 

/home/ec2-user/tf-notebook/query_intent_detection/slotfilling/data/load.pyc in atisfull()
     49 
     50     try:
---> 51         train_set, test_set, dicts = pickle.load(f)
     52     except UnicodeDecodeError:
     53         train_set, test_set, dicts = pickle.load(f, encoding='latin1')

UnpicklingError: invalid load key, '<'.
```

**Comment by chsasank** [comment]
Tried with python3?

**Comment by iuria21** [comment]
No, I tried with python2.7. Nevertheless, I think that the problem was in the atis.pkl file, it seems that it didn't download it correctly. I solved the problem right now downloading the atis.pkl by other link, because when I go to (https://www.dropbox.com/s/3lxl9jsbw0j7h8a/atis.pkl?dl=0) it says that the file is not currently there.

I don't know, but, in any case, I solved the problem, and thanks for the early answer!
